Somehow spinlock implemetation was not working on iOS. So changed locking
from spinlock to mutex.
Not required for Linux.
Maybe required for Android. Found similar dicussion on forum.

diff --git a/include/libcap_internal_user.h b/include/libcap_internal_user.h
index 1144994..fab2061 100644
--- a/include/libcap_internal_user.h
+++ b/include/libcap_internal_user.h
@@ -17,6 +17,12 @@
 #include <string.h>
 #include <errno.h>
 
+#define pthread_spinlock_t pthread_mutex_t
+#define pthread_spin_lock pthread_mutex_lock
+#define pthread_spin_unlock pthread_mutex_unlock
+#define pthread_spin_init(a,b) pthread_mutex_init(a, NULL)
+#define pthread_spin_destroy pthread_mutex_destroy
+
 /**
  * Mutex support.  Follow kernel return convention.
  */
